json_minify/test_json_minify.py

```diff
@@ -83,5 +125,28 @@ def test_3(self):
     def test_4(self):
         self.template(4)
 
+    def _space_template(self, in_string, expected):
+        actual = json_minify(in_string, strip_space=False)
+        self.assertEqual(actual, expected)
+
+    def test_space_keeping(self):
+        self._space_template('', '')
+        self._space_template('\n', '\n')
+        self._space_template(' \r\n ', ' \r\n ')
+        self._space_template('\t', '\t')
```

Comment:
Ah... In that case, can we drop all of these and merge `_space_template` into `template`?